scripts/generate-components-handlers/utils.ts

```diff
@@ -0,0 +1,42 @@
+import * as ts from "typescript";
+
+interface CompilerSourceFile {
+  fileName: string;
+  content: string;
+  sourceFile?: ts.SourceFile;
+}
+
+export const createCompilerHostForFile = (
```

Comment:
maybe rename file to reflect it's real content ? something like `mockCompilerHost` ?

Reply:
Hmmm not sure about mock since it sounds like its intended for testing.